### PR TITLE
Ubuntu Azure kernels: Add review stage and superseded status in DB

### PR DESCRIPTION
**Pull request by stefanlupsa**
* Add report into database that the kernel has been picked up for review, ask for confirmation from user.
* Add superseded status, keep true,false values for database insert
* Change kernel sign-off user input to choice parameter 
* Refactor sign-off mail

### CODE REVIEW

File: linux_pipeline/Jenkinsfile_ubuntu_azure_kernel_validation [hunk 1]
```diff
@@ -114,6 +114,42 @@ EOF
...
+def sendSignOffMail(mailType,warningMessage) {
+    def kernelAndDistroID = "${env.KernelType} ${env.KernelVersion} kernel for ${env.distro}"
+    def SIGNOFF_EMAIL_SUBJECT_HASH = [
+        'true' : "was reviewed by ${env.SIGNED_OFF_BY}",
+        'false' : "was reviewed by ${env.SIGNED_OFF_BY}",
+        'Superseded' : "was reviewed by ${env.SIGNED_OFF_BY}",
+        'timeout' : "was not signed off in time."]
+    def SIGNOFF_EMAIL_MESSAGE_HASH = [
+        'true' : "is OK and was signed off by ${env.SIGNED_OFF_BY}",
+        'false' : "is not ok / can't signed off. Reviewed by ${env.SIGNED_OFF_BY}",
```
Comment (ader1990) [inline linux_pipeline/Jenkinsfile_ubuntu_azure_kernel_validation]: cannot be*

File: linux_pipeline/Jenkinsfile_ubuntu_azure_kernel_validation [hunk 2]
```diff
@@ -114,6 +114,42 @@ EOF
...
+        'Superseded' : "was superseded during tesing. Reviewed by ${env.SIGNED_OFF_BY}",
```
Comment (ader1990) [inline linux_pipeline/Jenkinsfile_ubuntu_azure_kernel_validation]: testing*

File: linux_pipeline/Jenkinsfile_ubuntu_azure_kernel_validation (hunk 1 above)
Comment (stefanlupsa) [reply linux_pipeline/Jenkinsfile_ubuntu_azure_kernel_validation]: done

File: linux_pipeline/Jenkinsfile_ubuntu_azure_kernel_validation (hunk 2 above)
Comment (stefanlupsa) [reply linux_pipeline/Jenkinsfile_ubuntu_azure_kernel_validation]: done

**Pull request merged**